frontend/src/components/organizations/NotForProfitQuestions.vue

```diff
@@ -0,0 +1,175 @@
+              <v-radio label="No" :value="0"></v-radio>
+            </v-col>
+          </v-row>
+        </v-radio-group>
+      </v-card>
+
+      <v-card variant="outlined" class="soft-outline px-2 py-6 my-4 w-100">
+        <AppLabel>Are the majority of the elected board members full time BC residents?</AppLabel>
+        <v-radio-group v-model="model.boardMembersResidentsOfBC" :readonly="false" :rules="[...rules.required]">
+          <v-row no-gutters>
+            <v-col cols="12" sm="2" lg="1">
+              <v-radio label="Yes" :value="1"></v-radio>
+            </v-col>
+            <v-col cols="12" sm="2" lg="1">
+              <v-radio label="No" :value="0"></v-radio>
+            </v-col>
+          </v-row>
+        </v-radio-group>
+      </v-card>
+    </v-container>
+  </v-card>
+</template>
+
+<script>
+import AppLabel from '@/components/ui/AppLabel.vue'
+import AppDateInput from '@/components/ui/AppDateInput.vue'
+import AppMissingInfoError from '@/components/ui/AppMissingInfoError.vue'
+import alertMixin from '@/mixins/alertMixin'
+import rules from '@/utils/rules'
+import AppDocumentUpload from '@/components/ui/AppDocumentUpload.vue'
+import moment from 'moment'
+import { DOCUMENT_TYPES, DOCUMENT_LABELS, APPLICATION_ERROR_MESSAGES } from '@/utils/constants'
+import { isEmpty, cloneDeep } from 'lodash'
+
+export default {
+  components: { AppLabel, AppDateInput, AppDocumentUpload, AppMissingInfoError },
+  mixins: [alertMixin],
+  props: {
+    organization: {
+      type: Object,
+      required: true,
+      default: () => {
+        return {}
+      },
+    },
+    loading: {
+      type: Boolean,
+      default: false,
+    },
+  },
+  emits: ['update', 'deleteDocument', 'documentsComplete'],
+  data() {
+    return {
+      choice: undefined,
+      selectedDate: undefined,
+      model: undefined,
+      todayDate: new Date().toISOString().slice(0, 10),
+    }
+  },
+  computed: {
+    isReferenceLetterRequired() {
+      const pastDate = moment().subtract(4, 'years').startOf('day')
+      const modelDate = moment(this.model?.dateOfIncorporation).startOf('day')
+      return pastDate < modelDate
+    },
+    areDocumentsComplete() {
+      if (this.isReferenceLetterRequired) {
+        return !isEmpty(this.model.documentsToUpload) || !isEmpty(this.model.uploadedDocuments)
+      }
+      return true
+    },
+  },
+  watch: {
+    model: {
+      handler(value) {
+        this.$emit('update', value)
+        this.$emit('documentsComplete', this.areDocumentsComplete)
+      },
+      deep: true,
+    },
+  },
+  async created() {
+    this.rules = rules
+    this.DOCUMENT_TYPES = DOCUMENT_TYPES
+    this.DOCUMENT_LABELS = DOCUMENT_LABELS
+    this.APPLICATION_ERROR_MESSAGES = APPLICATION_ERROR_MESSAGES
+    this.model = cloneDeep(this.organization)
+  },
+  methods: {
+    deleteUploadedDocument(documentId) {
+      const foundDoc = this.model.uploadedDocuments.findIndex((el) => el.documentId === documentId)
```

Comment:
it would be great if we can add a check if foundDoc > -1

Reply:
done